src/redux/preview.js

```diff
@@ -51,7 +52,8 @@ module.exports.previewReducer = (state, action) => {
         return module.exports.getInitialState();
     case 'SET_PROJECT_INFO':
         return Object.assign({}, state, {
-            projectInfo: action.info
+            projectInfo: action.info ? action.info : {},
+            projectNotAvailable: !!action.info
```

Comment:
Maybe I'm not following, but doesn't this do the opposite of what we want? If `action.info` is a bunch of project data, won't `!!action.info` be `true`? As in, we'd think the project is not available?